teuthology/config.py

```diff
@@ -285,10 +284,13 @@ def set_config_attr(obj):
 
 
 def _get_config_path():
+    config_path = os.environ.get('TEUTHOLOGY_CONFIG', '~/.teuthology.yaml')
+    config_path = os.path.join(os.path.expanduser(config_path))
```

Comment:
Bad style, overriding variable.

Reply:
Also if we declare the env for the config, we should stop immediately if such file does not exists.